index.html

```diff
@@ -86,7 +77,12 @@ <h2>Leave a Message</h2>
     <h2>Messages</h2>
     <ul></ul>
   </section>
-  <footer></footer>
+  <footer>
+    <section id="connect">
+
+      <h2>Connect</h2><a href="https://github.com/Jessica12492/intro-to-programming-dorado">Github</a><a
+        href="https://mail.google.com">Gmail</a><a href="https://www.facebook.com">Facebook</a>
```

Comment:
These are all kinda squished together right now; would be good to have more spacing between each `a` tag so that it's more easily distinguishable for the user:

![Screen Shot 2023-01-31 at 2 00 52 PM](https://user-images.githubusercontent.com/9094098/215856890-53810530-7a50-43b6-8df1-26a7537245e4.png)

Could do this with some margin styles for example:
```css
#connect a {
    margin-left: 20px;
}
```